test/witan/models/acceptance/workspace_test.clj

```diff
@@ -36,7 +36,7 @@
    :project-births             {:var #'witan.models.dem.ccm.fert.fertility-mvp/project-births-from-fixed-rates}
    :combine-into-births-by-sex {:var #'witan.models.dem.ccm.fert.fertility-mvp/combine-into-births-by-sex
                                 :params {:proportion-male-newborns (double (/ 105 205))}}
-   :project-asmr         {:var #'witan.models.dem.ccm.mort.mortality-mvp/project-asmr
+   :project-asmr-average-fixed {:var #'witan.models.dem.ccm.mort.mortality-mvp/project-asmr-average-fixed
```

Comment:
in reference to previous comment - it needs to change on the right, but not the left